charts/azure-k8s-metrics-adapter/README.md

```diff
@@ -0,0 +1,66 @@
+
+## Installing the Chart
+
+```sh
+helm install --name my-release charts/azure-k8s-metrics-adapter
+```
+
+## Uninstalling the Chart
+
+```sh
+helm delete my-release
+```
+
+## Configuration
+
+### All Available Configuration Values
+
+| Parameter | Description | Default |
+| --------- | ----------- | ------- |
+| `image.repository` | Image repository | `jsturtevant/azure-k8-metrics-adapter-amd64` |
+| `image.tag` | Image tag | `latest` |
+| `image.pullPolicy` | Image pull policy | `Always` |
+| `logLevel` | Log level for V logs | `2` |
+| `replicaCount`  | Number of azure-k8s-metrics-adapter replicas  | `1` |
+| `adapterSecurePort` | Port on which the adapter is listening | `6443` |
+| `apiServiceInsecureSkipTLSVerify` | Disables TLS certificate verification when communicating with the apiService | `true` |
+| `apiServiceGroupPriorityMinimum` | The priority the APIService group should have at least | `100` |
+| `apiServiceVersionPriority` | Controls the ordering of this API version inside of its group | `100` |
+| `azureAuthentication.method` | Which method to use when authenticating with the Azure Resource Monitory service. Valid options are `msi`, `clientSecret`, and `clientCertificate` | `msi` |
+| `azureAuthentication.clientID` | The Azure Active Directory Application client ID to use if using the authentication method `clientSecret` or `clientCertificate` | `''` |
+| `azureAuthentication.tenantID` | Specifies the Tenant to which to authenticate if using the authentication method `clientSecret` or `clientCertificate | `''` |
+| `azureAuthentication.clientSecret` | Specifies the app secret to use if using the authentication method `clientSecret` | `''` |
+| `azureAuthentication.clientCertificate` | Specifies the contents of the certificate if using the authentication method `clientCertificate`  | `''` |
+| `azureAuthentication.clientCertificatePath` | Specifies certificate Path to use if using the authentication method `clientCertificate`  | `''` |
+| `azureAuthentication.azureClientCertificatePassword` | Specifies the certificate password to use  if using the authentication method `clientCertificate`  | `''` |
+| `defaultSubscriptionId` | Specifies the subscription to use instead of using Azure Instance Metadata  | `''` |
+| `extraArgs` | Optional flags for azure-k8s-metrics-adapter | `{}` |
+| `extraEnv` | Optional environment variables for azure-k8s-metrics-adapter | `{}` |
```

Comment:
Maybe provide more information about what is expected for `extraArgs` & `extraEnv`, if we should even provide this?

Reply:
I can drop these, for the configuration that is required for the running of the application I don't believe these are needed. @jsturtevant thoughts?

Reply:
They do not believe env is needed if we cover the basics we know about it.  The option to add args might be nice, tough there aren't many now.

Reply:
I think it would be good to leave them unless there is a demand maybe? Otherwise we need to keep on supporting this :)